ngx_lua_datadog_TEMPLATE.rockspec

```diff
@@ -0,0 +1,18 @@
+package = "ngx_lua_datadog"
```

Comment:
So here's the deal -- rockspecs are *required* to have the version stamped into the filename, as well as in the file itself.  IMO this is both stupid and annoying, but it is what it is.  So I figured it was best to make a template file and a script to `sed` and copy it to have the version in all the right places...